build.properties

```diff
@@ -76,6 +76,18 @@ behat.options.strict = false
 # shows exception backtraces, 3 shows debugging information.
 behat.options.verbosity = 2
 
+# The location where to save the screenshots. If empty, the screenshot will not
+# be created.
+# Example:
+# behat.screenshots_dir = ${tmp.dir}/behat
+behat.screenshots_dir =
+
+# The AWS S3 URI location where to upload the Behat screenshot artifacts. This
+# property is disabled by default. You'll need to explicit configure it in the
+# continuous integration system.
+# Example:
+# behat.artifacts_s3_uri = s3://joinup2/cphp-behat-artifacts
```

Comment:
I wouldn't hardcode it to our S3 bucket, there is no value in that, just provide an example path like `s3://mybucket/mypath`.